seedlet/search.php

```diff
@@ -42,8 +42,8 @@
 				// End the loop.
 			endwhile;
 
-			// Previous/next page navigation.
-			seedlet_the_posts_navigation();
+			// Numbered pagination.
+			seedlet_the_posts_pagination();
```

Comment:
Should we keep this also on Spearhead? I think we should but maybe @kjellr has some input about this?

Reply:
"This" being the numbered navigation on the search results page

Reply:
Yeah, we should keep the numbered pagination on the search results and archive pages. Just the prev/next post nav from single post pages should be removed in Spearhead.

Reply:
then ⛵